scripts/load_safe_deployments.py

```diff
@@ -0,0 +1,44 @@
+import json
+import os
+import shutil
+
+from git import Repo
+
+GIT_URL = "https://github.com/safe-global/safe-deployments.git"
+SAFE_DEPLOYMENTS_VERSION = "v1.27.0"  # safe-deployments tag version
+REPO_DIR = "safe-deployments"  # temporary folder to clone the repo
+DEPLOYMENTS_FOLDER = REPO_DIR + "/src/assets"  # folder where are the deployments
+SAFE_ETH_PY_DEPLOYMENTS_PATH = "../gnosis/safe/safe_deployments.py"  # Path and filename where the new dict is being stored
+
+
+def clean_resources():
```

Comment:
Not required, but you could use a custom context manager to make sure resources are always cleaned: https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager